src/editor-sidebar/CanvasItem.js

```diff
@@ -1,6 +1,7 @@
 /** @jsxImportSource @emotion/react */
 
 import { useState } from 'react';
+import { Btn } from '../primitives/Btn';
```

Comment:
Here is an opinion of mine (which means it can be ignored): Why not called it `Button`?